provider/resources.go

```diff
@@ -115,6 +101,18 @@ func Provider() tfbridge.ProviderInfo {
 
 	prov.MustComputeTokens(tfbridgetokens.SingleModule("dnsimple_", mainMod,
 		tfbridgetokens.MakeStandard(mainPkg)))
+
+	prov.P.ResourcesMap().Range(func(key string, value shim.Resource) bool {
```

Comment:
This would be a great place to link to the bridge issue and provide some background on why this is necessary.

Reply:
Done.